src/Sylius/Bundle/PayumBundle/Action/ConvertPaymentToPaypalExpressAction.php

```diff
@@ -67,6 +67,14 @@ public function execute($request)
             $details['L_PAYMENTREQUEST_0_NAME'.$m] = 'Shipping Total';
             $details['L_PAYMENTREQUEST_0_AMT'.$m]  = round($shippingTotal / 100, 2);
             $details['L_PAYMENTREQUEST_0_QTY'.$m]  = 1;
+
+            $m++;
+        }
+
+        if (0 !== $paymentTotal = $order->getAdjustmentsTotal(AdjustmentInterface::PAYMENT_ADJUSTMENT)) {
+            $details['L_PAYMENTREQUEST_0_NAME'.$m] = 'Payment Fee Total';
+            $details['L_PAYMENTREQUEST_0_AMT'.$m]  = round($paymentTotal / 100, 2);
```

Comment:
We do not have this configuration on currencies yet. :( But we will definitely look into this issue.